app/assets/javascripts/discourse/app/components/edit-category-tab.js

```diff
@@ -25,23 +29,32 @@ export default Component.extend({
     scheduleOnce("afterRender", this, this._addToCollection);
   },
 
+  willDestroy() {
```

Comment:
Usually we use `willDestroyElement` for components. Unless there's a reason that this needs to be `willDestroy` specifically.